tests/method-chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -544,7 +570,7 @@ d3.select('body')
   .at({ width: 30, fill: '#f0f' })
   .st({ fontWeight: 600 })
 
-d3.scaleLinear()
+const myScale = d3.scaleLinear()
   .domain([1950, 1980])
   .range([0, width])
```

Comment:
Why not keep the old one, add a new one?

Reply:
Because the old one was not real-life code. That's not how d3 is used. Our heuristics are developed to format real code. Ideally all the foo-bar-baz should be removed from that folder and replaced with more reasonable examples.